make/photon/prepare/templates/docker_compose/docker-compose.yml.jinja

```diff
@@ -378,6 +378,9 @@ services:
     volumes:
       - {{data_volume}}/chart_storage:/chart_storage:z
       - ./common/config/chartserver:/etc/chartserver:z
+{% if gcs_keyfile %}
```

Comment:
This attribute is not reliable:
https://github.com/goharbor/harbor/blob/master/make/photon/prepare/utils/docker_compose.py#L37

You should make sure this attribute is set only when the storage is `"gcs"`